bindings/nodejs/examples/wallet/06-send-micro-transaction.ts

```diff
@@ -30,7 +30,7 @@ async function run() {
         await account.sync();
 
         console.log(
-            `Sending '${SEND_MICRO_AMOUNT}' coin(s) to '${RECV_ADDRESS}'...`,
+            `Sending '${SEND_MICRO_AMOUNT.toString()}' coin(s) to '${RECV_ADDRESS}'...`,
```

Comment:
You're right, it's not needed, the `console.log` function calls it by default, I was afraid that it will add `n` to the number if `toString()` was not called, but it actually doesn't.